ui/components/multichain/account-list-item/account-list-item.js

```diff
@@ -122,9 +131,6 @@ const AccountListItem = ({
 
   const useBlockie = useSelector(getUseBlockie);
   const { isEvmNetwork } = useMultichainSelector(getMultichainNetwork, account);
-  const setAccountListItemMenuRef = (ref) => {
-    setAccountListItemMenuElement(ref);
-  };
```

Comment:
Removed using `useState` to set refs.
UseState incurs an additional render cycle for when the element ref is set.

I wonder if there are other places where we use `useState` to set element references.